<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageBooks</class>
 <widget class="QDialog" name="ManageBooks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manage Books</string>
  </property>
  <widget class="QPushButton" name="AddBook">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Book</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EditBook">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>110</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Book</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteBook">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>190</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Book</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FindBook">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>270</y>
     <width>100</width>
     <height>75</height>
    </rect>
   </property>
   <property name="text">
    <string>Find Book</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
